src/components/form/form.style.js

```diff
@@ -61,26 +72,31 @@ export const StyledForm = styled.form`
       height: ${height};
     `}
 
-  & ${StyledFormField}, ${StyledFieldset}, ${FieldsetStyle}, > ${StyledButton} {
-    margin-top: 0;
-    margin-bottom: ${({ fieldSpacing, theme }) =>
-      theme.spacing * fieldSpacing}px;
-  }
+  ${({ fieldSpacing }) =>
+    css`
+      &
+        ${StyledFormField},
```

Comment:
Something is wrong with formatting here

Reply:
The formatting was checked, it turned out, that the prettier plugin formats code as above, so it's correct.